config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,37 @@
+if defined?(Rails::Server) && !Rails.env.test?
+  require 'rack/attack'
+
+  # The `rack/attack/railtie.rb` creates a `initializer` call, which is loaded
+  # too late because of the `:require => false` in our Gemfile, so this needs
+  # to be done manually.
+  Rails.application.middleware.use(Rack::Attack)
+
+  api_login_limit  = proc { ::Settings.server.rate_limiting.api_login.limit }
+  api_login_period = proc { ::Settings.server.rate_limiting.api_login.period.to_i_with_method }
+  request_limit    = proc { ::Settings.server.rate_limiting.request.limit }
+  request_period   = proc { ::Settings.server.rate_limiting.request.period.to_i_with_method }
+  ui_login_limit   = proc { ::Settings.server.rate_limiting.ui_login.limit }
+  ui_login_period  = proc { ::Settings.server.rate_limiting.ui_login.period.to_i_with_method }
+
+  # Throttle all requests by IP
+  #
+  # Key: "rack::attack:#{Time.now.to_i/:period}:req/ip:#{req.ip}"
+  Rack::Attack.throttle('req/ip', :limit => request_limit, :period => request_period) do |req|
+    req.ip # unless req.path.start_with?('/assets')
```

Comment:
Is this comment still required?

Reply:
Came as part of the default config that I copied this from:

https://github.com/rack/rack-attack/blob/6-stable/docs/example_configuration.md

I can get rid of it if you want.

Reply:
Ahh I see - let's get rid of it, otherwise it's confusing to a reader as it looks like a TODO decision that was never made.

Reply:
Fixed